go/main.go

```diff
@@ -118,9 +123,29 @@ func main() {
 	}
 
 	updatedResourceDisplayName := resourceDisplayName + "-new-name"
+	updatePublicNetworkAccessValues := iotcentral.PossiblePublicNetworkAccessValues()[1] // publicNetworkAccessEnabled
```

Comment:
instead of putting publicNetworkAccessEnabled in the comments maybe just rename the variable to be publicNetworkAccessEnabled or enabled